controllers/controllers/workloads/cfbuild_controller.go

```diff
@@ -60,6 +60,13 @@ func NewCFBuildReconciler(client CFClient, scheme *runtime.Scheme, log logr.Logg
 //+kubebuilder:rbac:groups=korifi.cloudfoundry.org,resources=buildworkloads,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=korifi.cloudfoundry.org,resources=buildworkloads/status,verbs=get
 
+//+kubebuilder:rbac:groups=korifi.cloudfoundry.org,resources=buildreconcilerinfos,verbs=get;list;watch;create;update;patch;delete
```

Comment:
It is pretty much possible to be missing something, but I don't get why removing HNC needs changes in the buildreconcilerinfos/clusterbuilders RBAC permissions

Reply:
We tried removing these "extra" rbac annotations, but they end up being necessary. They're necessary since a service account can't create rolebindings that grant RBAC abilities that it doesn't already have. I moved the annotations around and added a clear comment explaining their purpose. In the long-run, we may want to find an automated way to ensure that the service account has all permissions granted by all cf_roles, but that's a large chunk of work.